sycl/test/basic_tests/memory-consumption.cpp

```diff
@@ -0,0 +1,70 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_BE=PI_LEVEL0 %t.out
```

Comment:
That will not work on machines without OpenCL or Level0 RTs.
please update RUN lines to make them functionol in all possible configurations

Reply:
done